src/components/Search/search.js

```diff
@@ -5,7 +5,7 @@ import { Icon, Button, Loader} from 'semantic-ui-react'
 
 import { studentOptions, facultyOptions, whoami } from '../../actions/index'
 import { urlStudentQuery, urlFacultyQuery, urlInterestQuery, urlProfile, } from '../../urls'
-
+import { debounce } from 'lodash'
```

Comment:
Shift this line to the top along with other imports. As per omniport conventions, you should put the imports from foreign libraries on the top followed by imports from the home directory and then the other model imports.